qs-common/qs-common-model/src/main/java/hu/icellmobilsoft/quarkus/sampler/model/jpatest/SampleEntityProjection.java

```diff
@@ -0,0 +1,33 @@
+/*-
+ * #%L
+ * Sampler
+ * %%
+ * Copyright (C) 2022 i-Cell Mobilsoft Zrt.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package hu.icellmobilsoft.quarkus.sampler.model.jpatest;
```

Comment:
A projection-öket másik package-be raknám, inkább a `model` mellé vagy alá, vagy még inkább a `query`-khez közel, mert azok definiálják az egyes projection-öket.